### DISCUSSION TITLE
Shower Prison

### DISCUSSION THREAD

**Discussion by Gloktober** [winstxnhdw/lc-hax#90] [Ideas]
// ShowerTrigger
// Token: 0x06000ED4 RID: 3796 RVA: 0x0007CCEE File Offset: 0x0007AEEE
private void AddPlayerToShower(PlayerControllerB playerScript)
{
	if (!this.playersInShower.Contains(playerScript))
	{
		Debug.Log(string.Format("Added player #{0} to shower", playerScript.playerClientId));
		this.playersInShower.Add(playerScript);
	}

Could this be used to create a '' prison '' for troll reasons ? like you could spawn a shower and a locker to close the shower and tp some players in it lol

**Comment by winstxnhdw** [comment]
LMFAO, sounds great. I'll try it.

**Reply by Gloktober** [reply]
awesome ! Give me some feedback on it ;)

**Reply by winstxnhdw** [reply]
To be honest, teleporting the cupboard on the person is enough to restrict their movements. But I guess a shower would make them more visible.

**Reply by winstxnhdw** [reply]
For now, you can just do `/tp <player> 0 10000 0` to send them to the void lol

**Reply by Gloktober** [reply]
yea exactly ! Would be funnier to trap them in a shower while in the ship for the fun of it

**Reply by winstxnhdw** [reply]
They'd be able to store the items if they are in the ship, though.

**Reply by Gloktober** [reply]
lol that tp is hilarious
